boards/arduino-mega2560/src/lib.rs

```diff
@@ -22,7 +22,17 @@ pub use crate::hal::spi;
 pub use crate::hal::adc;
 
 pub type Delay = crate::hal::delay::Delay<hal::clock::MHz16>;
-pub type Serial<IMODE> = crate::hal::usart::Usart0<hal::clock::MHz16, IMODE>;
+
+pub mod usart {
+    pub use atmega2560_hal as hal;
+    pub use avr_hal_generic::serial::*;
+
+    pub type Usart0<IMODE> = crate::hal::usart::Usart0<hal::clock::MHz16, IMODE>;
+    pub type Usart1<IMODE> = crate::hal::usart::Usart1<hal::clock::MHz16, IMODE>;
+    pub type Usart2<IMODE> = crate::hal::usart::Usart1<hal::clock::MHz16, IMODE>;
+    pub type Usart3<IMODE> = crate::hal::usart::Usart1<hal::clock::MHz16, IMODE>;
```

Comment:
You forgot to update these here:

```suggestion
    pub type Usart2<IMODE> = crate::hal::usart::Usart2<hal::clock::MHz16, IMODE>;
    pub type Usart3<IMODE> = crate::hal::usart::Usart3<hal::clock::MHz16, IMODE>;
```

Reply:
Done